measCompApp/Db/measCompLongOut.template

```diff
@@ -3,15 +3,15 @@
 ###################################################################
 record(longout, "$(P)$(R)")
 {
-    field(PINI, "YES")
+#   field(PINI, "YES")
     field(DTYP, "asynUInt32Digital")
     field(OUT,  "@asynMask($(PORT),$(ADDR),$(MASK))DIGITAL_OUTPUT")
 }
 
 record(longin, "$(P)$(R)_RBV")
 {
     field(DTYP, "asynUInt32Digital")
-    field(INP,  "@asynMask($(PORT),$(ADDR),$(MASK))DIGITAL_OUTPUT")
+    field(INP,  "@asynMask($(PORT),$(ADDR),$(MASK))DIGITAL_INPUT")
```

Comment:
Changing DIGITAL_OUTPUT to DIGITAL_INPUT is incorrect.  DIGITAL_OUTPUT is a readback of the output register.  DIGITAL_INPUT is reading the input register, which is different. Did you observe and issue that motivated this change?

Reply:
reverted back